README.rst

```diff
@@ -2,6 +2,13 @@
 MLflow: A Machine Learning Lifecycle Platform
 =============================================
 
+MLflow 2.0
+----------
+
```

Comment:
"Please help us test MLflow 2.0 by installing the MLflow 2.0 RC. "

Can we also include a link to the docs?

Reply:
<img width="1030" alt="image" src="https://user-images.githubusercontent.com/17039389/199385757-50f7fbb8-c1a3-4847-956e-3d2b07fe396b.png">

https://github.com/harupy/mlflow/tree/readme-for-2.0